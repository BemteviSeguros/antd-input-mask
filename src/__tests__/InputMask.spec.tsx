import React from "react";
import { render } from "@testing-library/react";

import { InputMask } from "../component";

it("correctly renders", () => {
  const { container } = render(<InputMask mask="01111-010" />);
  expect(container).toMatchSnapshot();
  expect(container.getElementsByClassName("ant-input").length).toBe(1);
});
