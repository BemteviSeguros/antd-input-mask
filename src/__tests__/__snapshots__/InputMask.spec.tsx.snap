// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders 1`] = `
<div>
  <input
    class="ant-input"
    type="text"
    value=""
  />
</div>
`;
